data.json

```
@@ -1084,6 +1084,16 @@
             ],
             "description": "Component framework that runs at build time, converting your components into highly efficient imperative code that surgically updates the DOM."
         },
+        {
+            "name": "Devopness",
+            "link": "https://github.com/devopness/devopness",
+            "label": "good first issue",
+            "technologies": [
+                "JavaScript",
+                "TypeScript"
+            ],
+            "description": "Devopness aims to drastically simplify the way we manage cloud applications and multi cloud infrastructure in a secure and productive way."
```

```suggestion
            "technologies": [
                "TypeScript"
            ],
            "description": "Devopness aims to drastically simplify the way we manage cloud applications and multi cloud infrastructure in a secure and productive way."
```
I reckon this should just be TypeScript as whilst TypeScript is built on top of JavaScript, the repository is almost 100% TypeScript.

---

Thanks for the suggestion @sammyhori 

I agree with you that we can keep only TypeScript.